tests/ats/.pre-commit-config.yaml

```diff
@@ -0,0 +1,70 @@
+minimum_pre_commit_version: 1.21.0
```

Comment:
Shouldn't this come from `giantswarm/github`?

Reply:
It should, we have a ticket for that: https://github.com/giantswarm/giantswarm/issues/19630